sys/linux/socket_tipc_netlink.txt

```diff
@@ -170,6 +170,8 @@ tipc_nl_media_policy [
 tipc_nl_node_policy [
 	TIPC_NLA_NODE_ADDR	nlattr[TIPC_NLA_NODE_ADDR, int32]
 	TIPC_NLA_NODE_UP	nlattr[TIPC_NLA_NODE_UP, void]
+	TIPC_NLA_NODE_ID	nlattr[TIPC_NLA_NODE_ID, array[int8]]
```

Comment:
For TIPC_NLA_NODE_ID, can we do better than `array[int8]`?  Does this value correspond to some other value that is (or should be) described too?

Reply:
It seems that this value is not returned by any syzcall. I tried to create a resource for this but it seems that resource can't be an array so I just set it to be array[int8} for now. Any suggestions?